pkg/api/api.proto

```diff
@@ -513,3 +513,28 @@ message OptionalBool {
 message OptionalFileMode {
   uint32 value = 1;
 }
+
+// CompoundFieldOwners tracks 'plugin ownership' of compound fields, typically
+// maps or slices.
```

Comment:
key: plugin?

value: key of map or index of slice?

Reply:
Key identifies owned entry, value is the owning plugin. Updated comments to clarify this.